pkg/cloud/amazon/ecr_test.go

```diff
@@ -25,7 +29,40 @@ func TestCreateNewSessionWithRegionFromAwsRegion(t *testing.T) {
 	os.Setenv("AWS_REGION", "us-east-1")
 	os.Setenv("AWS_DEFAULT_REGION", "")
 	os.Setenv("AWS_PROFILE", "")
-	sess, err := amazon.NewAwsSessionWithoutOptions()
+	sess, err := session.NewAwsSessionWithoutOptions()
 	assert.Nil(t, err)
 	assert.Equal(t, "us-east-1", *sess.Config.Region)
 }
+
+func restoreHome(oldHome string) {
+	os.Setenv("HOME", oldHome)
+}
+
+func switchHome() (string, error) {
```

Comment:
don't we have something like this in our current testhelpers? If not, maybe this should be moved into a more generic package to be more readily reusable by other tests?